torch/utils/cpp_extension.py

```diff
@@ -573,6 +573,14 @@ def build_extensions(self) -> None:
                         extension.extra_compile_args[ext] = []
 
             self._add_compile_flag(extension, '-DTORCH_API_INCLUDE_EXTENSION_H')
+
+            if extension.py_limited_api:
+                 # compile any extension that has passed in py_limited_api to the
+                 # Extension constructor with the Py_LIMITED_API flag set to our
+                 # min supported CPython version, here, 3.9.
+                 # See https://docs.python.org/3/c-api/stable.html#c.Py_LIMITED_API
+                self._add_compile_flag(extension, '-DPy_LIMITED_API=0x03090000')
```

Comment:
I feel like this is the 3rd time I see you hardcoding this constant (0x03090000), let's put it somewhere and re-use :) 
It will make upgrading it easier!

Reply:
This is the only usage in our pytorch/pytorch repo tho :p

Reply:
It is our lowest supported version, I would expect we do have some checks that people don't run below this?
But yeah I would prefer users of cpp extensions to use our constant value rather than hardcoding theirs. To get the lower version upgrade when we do it.